CHANGELOG.md

```diff
@@ -2,13 +2,15 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/55654104/114203250-46dd1d80-9958-11eb-9f59-e3445c0ff9f7.png)
I don't really know if I like the 50% selection in the line to the next circle at scenarios where the previous circle isn't orange. I think it's good enough for now, but maybe we can improve this somehow? Let's have a meeting for this as well.